sol/main.c

```diff
@@ -509,6 +509,19 @@ cleanup_sol(struct sol_config *sol_conf)
 	return 0;
 }
 
+int
+sol_conf_put(struct sol_config *sol_conf)
+{
+	/*
+	 * Atomically decrements the atomic counter (v) by one and returns true
```

Comment:
What is "v" here?

Reply:
I just dropped the string " (v)" since it doesn't improve the comment. It was just referring to the atomic variable being used in the code.

I've copied that comment from `gk_conf_put()`, so I've added patch **gk: improve comments on reference counting** to fix that comment there as well.